tests/config.rs

```diff
@@ -5,15 +5,24 @@ use std::io::prelude::*;
 use std::process::{Child, Command, Stdio};
 
 #[test]
-fn it_generates_the_config() {
+fn it_generates_the_config_unix_eol() {
+    generate_config_with("\n");
+}
+
+#[test]
+fn it_generates_the_config_windows_eol() {
+    generate_config_with("\r\n");
+}
+
+fn generate_config_with(eol: &str) {
     let fake_home_dir = env::current_dir()
         .expect("could not retrieve cwd")
-        .join(".it_generates_the_config");
+        .join(format!(".it_generates_the_config_{}", random_chars(5)));
```

Comment:
because `cargo test` is multithreaded, I had to add this random_char thing to the end of the home directory name to keep the threads from deleting each other's home dir. We should probably enforce this for all tests that create directories in the future.